src/amo/components/SearchResults.js

```diff
@@ -72,18 +47,16 @@ class SearchResults extends React.Component {
       }
     }
 
-    const message = (
-      (<p ref={(ref) => { this.message = ref; }} className={classNames({
-        'visually-hidden': hideMessageText,
-        'SearchResults-message': !hideMessageText,
-      })}>{messageText}</p>)
-    );
-
     return (
       <div ref={(ref) => { this.container = ref; }} className="SearchResults">
-        {resultHeader}
-        {message}
-        {searchResults}
+        <AddonCard addons={results}>
```

Comment:
The mocks show a header/footer though, that's what those are, right?

<img width="594" alt="screenshot 2016-11-25 18 08 01" src="https://cloud.githubusercontent.com/assets/211578/20636855/7ed66448-b33a-11e6-81e2-6894277b28a2.png">

Admittedly I have not run this locally. How does it look?